IDEAS/Buildings/Data/Constructions/InsulatedFloorHeating.mo

```diff
@@ -1,9 +1,10 @@
 within IDEAS.Buildings.Data.Constructions;
 record InsulatedFloorHeating "Floor heating example with heat injection between PUR and screed"
   extends IDEAS.Buildings.Data.Interfaces.Construction(
-    locGain={2},
+    locGain={3},
     incLastLay = IDEAS.Types.Tilt.Floor,
-    mats={IDEAS.Buildings.Data.Materials.Concrete(d=0.10),
+    mats={IDEAS.Buildings.Data.Materials.EPC.GypsumPlasterForFinishing(d=0.02),
```

Comment:
This only holds if the floor is used between two zones (not on the ground floor). Maybe distinguish between `InsulatedFloor` and `InsulatedFloorOnGround`?

I would also remove or change the "FloorOnGround", as this was mentioned before in #1267 (concrete layer on top of the screed...)

Reply:
In the end we then have three floor models:
1. FloorOnGround
2. InsulatedFloorHeating
3. UninsulatedFloor

Shouldn't we change the terminology here to have some parallelism between the different constructions?
E.g. FloorOnGround, InsulatedFloor, and UninsulatedFloor? All construction layers can be combined with an embedded heating system, thus the `Heating` in the name seems obsolete to me.